virtcontainers/sandbox_test.go

```diff
@@ -1748,3 +1749,109 @@ func TestStartNetworkMonitor(t *testing.T) {
 	err = s.startNetworkMonitor()
 	assert.Nil(t, err)
 }
+
+func checkDirNotExist(path string) error {
+	if _, err := os.Stat(path); os.IsExist(err) {
+		return fmt.Errorf("%s is still exists", path)
+	}
+	return nil
+}
+
+func checkSandboxRemains() error {
+	var err error
+	if err = checkDirNotExist(sandboxDirConfig); err != nil {
+		return fmt.Errorf("%s still exists", sandboxDirConfig)
+	}
+	if err = checkDirNotExist(sandboxDirState); err != nil {
+		return fmt.Errorf("%s still exists", sandboxDirState)
+	}
+	if err = checkDirNotExist(configStoragePath); err != nil {
+		return fmt.Errorf("%s still exists", configStoragePath)
+	}
+	if err = checkDirNotExist(runStoragePath); err != nil {
+		return fmt.Errorf("%s still exists", runStoragePath)
+	}
+	if err = checkDirNotExist(path.Join("/sys/fs/cgroup/cpu/kata/", testSandboxID)); err != nil {
```

Comment:
It's generally not a good idea for tests to look at actual system files. I would create a variable (not a `const`) that the main code uses. That variable would build the path from `defaultCgroupParent`. Your test code can then redefine that variable so that the tests are not looking at `/sys`. For an example of what I'm talking about, see:

- https://github.com/kata-containers/runtime/blob/master/cli/kata-check_amd64_test.go#L52..L70

Reply:
thanks for your comments, I found that this part of code not needed, so I deleted directly in new PR~